tests/integration_tests.rs

```diff
@@ -260,6 +260,19 @@ fn sgx_get_att_quote() {
     run_test("sgx_get_att_quote", 0, None, None, None);
 }
 
+#[cfg(feature = "backend-kvm")]
+#[cfg(not(feature = "backend-sev"))]
```

Comment:
These save us some bytes in the compiled binary but this won't prevent someone from accidentally running this with the wrong backend.

Reply:
right